<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>553</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Mine Simulator (PJC)</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Desktop/s1_big_rock.png</normaloff>../../Desktop/s1_big_rock.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelLastStrike">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>220</y>
      <width>65</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ostatni strajk</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMonths">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>140</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>432</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMonthsText">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>120</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Months:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>300</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="nextMonthBtn">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>NextMonth</string>
    </property>
   </widget>
   <widget class="QFrame" name="frameBrown">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>331</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="FrBrToBlBtn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>170</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Move to B mine</string>
     </property>
    </widget>
    <widget class="QPushButton" name="minerListLeftBtn_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>41</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="disFrBrBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Dismiss</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSalary_3">
     <property name="geometry">
      <rect>
       <x>254</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>No data</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelName_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>No data</string>
     </property>
    </widget>
    <widget class="QLabel" name="bImg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/img/bru100.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="labelSalaryText_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Salary</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelProd_3">
     <property name="geometry">
      <rect>
       <x>284</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>No data</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelProdText_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Productivity</string>
     </property>
    </widget>
    <widget class="QPushButton" name="FrBrtoMBtn">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Move to M mine</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelStrikOp_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>No data</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelStrikOpText_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>100</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Strike Opportunity</string>
     </property>
    </widget>
    <widget class="QPushButton" name="minerListRightBtn_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>41</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>-&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelKids_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>No data</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelUnionText_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>140</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Trade Union</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelKidsText_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Kids</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelUnion_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>140</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>No data</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameNew">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>331</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="labelSalaryText">
     <property name="geometry">
      <rect>
       <x>46</x>
       <y>160</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Salary</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelProd">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>No data</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSalary">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>No data</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelKidsText">
     <property name="geometry">
      <rect>
       <x>46</x>
       <y>140</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Kids</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelStrikOp">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>140</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>No data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="minerListLeftBtn_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>41</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;-</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelUnion">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>160</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>No data</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelKids">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>140</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>No data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="newToUranBtn">
     <property name="geometry">
      <rect>
       <x>196</x>
       <y>190</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Hire to M mine</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelUnionText">
     <property name="geometry">
      <rect>
       <x>136</x>
       <y>160</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Trade Union</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelName">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>No data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="newToBrownBtn">
     <property name="geometry">
      <rect>
       <x>116</x>
       <y>190</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Hire to Br mine</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelProdText">
     <property name="geometry">
      <rect>
       <x>136</x>
       <y>120</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Productivity</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelStrikOpText">
     <property name="geometry">
      <rect>
       <x>136</x>
       <y>140</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Strike Opportunity</string>
     </property>
    </widget>
    <widget class="QPushButton" name="minerListRightBtn_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>120</y>
       <width>41</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>-&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="newToBlackBtn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Hire to B mine</string>
     </property>
    </widget>
    <widget class="QLabel" name="Icon">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/img/new.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameBlack">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>331</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="minerListLeftBtn_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>41</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;-</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelStrikOp_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>No data</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelKidsText_2">
     <property name="geometry">
      <rect>
       <x>46</x>
       <y>130</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Kids</string>
     </property>
    </widget>
    <widget class="QPushButton" name="minerListRightBtn_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>110</y>
       <width>41</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>-&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m1b3_3">
     <property name="geometry">
      <rect>
       <x>196</x>
       <y>180</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Hire to M mine</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelUnionText_2">
     <property name="geometry">
      <rect>
       <x>136</x>
       <y>150</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Trade Union</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSalary_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>No data</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelProdText_2">
     <property name="geometry">
      <rect>
       <x>136</x>
       <y>110</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Productivity</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelKids_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>No data</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelStrikOpText_2">
     <property name="geometry">
      <rect>
       <x>136</x>
       <y>130</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Strike Opportunity</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSalaryText_2">
     <property name="geometry">
      <rect>
       <x>46</x>
       <y>150</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Salary</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m1b2_3">
     <property name="geometry">
      <rect>
       <x>116</x>
       <y>180</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Hire to Br mine</string>
     </property>
    </widget>
    <widget class="QPushButton" name="disFrBlBtn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Dismiss</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelName_2">
     <property name="geometry">
      <rect>
       <x>46</x>
       <y>110</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>No data</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelProd_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>No data</string>
     </property>
    </widget>
    <widget class="QLabel" name="kImg">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>-10</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/img/kam100.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="labelUnion_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>No data</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameMixed">
    <property name="geometry">
     <rect>
      <x>369</x>
      <y>69</y>
      <width>331</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="mImg">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="res.qrc">:/img/ur100.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="labelStrikOp_4">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>140</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>No data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="minerListRightBtn_5">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>120</y>
       <width>41</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>-&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelKidsText_4">
     <property name="geometry">
      <rect>
       <x>56</x>
       <y>140</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Kids</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelUnionText_4">
     <property name="geometry">
      <rect>
       <x>146</x>
       <y>160</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Trade Union</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelProdText_4">
     <property name="geometry">
      <rect>
       <x>146</x>
       <y>120</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Productivity</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m1b3_5">
     <property name="geometry">
      <rect>
       <x>206</x>
       <y>190</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Dismiss</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSalary_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>No data</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelKids_4">
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>140</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>No data</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelStrikOpText_4">
     <property name="geometry">
      <rect>
       <x>146</x>
       <y>140</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Strike Opportunity</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m1b2_5">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>190</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Hire to Br mine</string>
     </property>
    </widget>
    <widget class="QPushButton" name="m1b1_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Hire to B mine</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSalaryText_4">
     <property name="geometry">
      <rect>
       <x>56</x>
       <y>160</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Salary</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelName_4">
     <property name="geometry">
      <rect>
       <x>56</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>No data</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelProd_4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>No data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="minerListLeftBtn_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>41</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;-</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelUnion_4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>160</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>No data</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelGameTitle">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Stencil</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>MINE SIMULATOR: BECOME A MINE BOSS!</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>closeBtn</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>554</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>565</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
